src/guides/language/evolution.md

```diff
@@ -37,6 +40,50 @@ environment:
 
 ## Changes in each release
 
+### Dart 3.2
+_Released 15 November 2023_
+| [Dart 3.2 announcement](https://medium.com/dartlang/)
+
+Dart 3.2 added enhancements to flow analysis, including:
+
+* Expanded [type promotion](/null-safety/understanding-null-safety#type-promotion-on-null-checks)
+  to work on private final fields. Previously only available for
+  local variables and parameters, now private final fields can promote to
+  non-nullable types through null checks and `is` tests. For example,
+  the following code is now sound:
+
+  ```dart
+  class Example {
+  final int? _privateField;
+  Example1(this._privateField);
+
+  f() {
+    if (_privateField != null) {
+      // _privateField has now been promoted; you can use it without
+      // null checking it.
+      int i = _privateField; // OK
+      }
+    }
+  }
+
+  // Private field promotions also work from outside of the class:
+  f(Example1 x) {
+    if (x._privateField != null) {
+      int i = x._privateField; // OK
+    }
+  }
+  ```
+  
+  For more information on when private final fields can and can't promote, check
+  out [Fixing type promotion failures](/tools/non-promotion-reasons).
+
+* Corrected inconsistencies in type promotion behvaior of
+  [if-case](/language/branches#if-case) statements
+  where the scrutinee throws an exception.
```

Comment:
I don't think we used the term "scrutinee" in our pattern matching docs, so likely want to avoid it here as Dart developers might not be familiar with it.

Taking a look at your branching and pattern docs, it seems you mostly referred to the scrutinee as a "value" and added context to explain. So perhaps "_where the value being matched_ throws an exception"?